docs/core/tutorials/cli-templates-create-template-pack.md

```diff
@@ -46,8 +46,11 @@ dotnet new console -n templatepack -o .
 
 The `-n` parameter sets the _.csproj_ filename to _templatepack.csproj_ and the `-o` parameters creates the files in the current directory. You should see a result similar to the following output.
 
+```dotnetcli
```

Comment:
@tdykstra How do you feel about these in the templates tutorial? The path in the command *does* let the user know what directory it should be invoked from. It's easy to get lost in these... It's one reason why I included it.

Reply:
> How do you feel about these in the templates tutorial?

What does "these" refer to?

Reply:
Sorry, the removal of the path in the command `C:\working` to indicate the command is run in the working folder.

Reply:
I like it better without the path because then I can use the Copy button in the snippet window. Also if you include the path you would need separate Windows and Linux snippet windows.